components/clipboard-button/index.js

```diff
@@ -15,12 +15,16 @@ import { findDOMNode, Component } from '@wordpress/element';
  */
 import { Button } from '../';
 
-class ClipboardButton extends Component {
+// This creates a container to put the textarea in which isn't removed by react
+// If react removes the textarea first, then the clipboard fails when trying to remove it
+class ClipboardContainer extends Component {
```

Comment:
Does this need to be a separate component, or can we assign the button of `ClipboardButton` as the `container`?

Reply:
It needs to have shouldUpdateFalse. The problem is if it gets redrawn, then the textarea that is inside gets removed, and the clipboard library throws an error when trying to remove it. We just need a component that can add things added to it, and isn't going to get recreated.

Reply:
I had it as the button originally, and kept getting an error in the clipboard library when it tried to remove the textarea, because react had already removed it because the button had been redrawn. You would get this error if you tried clicking on the button after you had already clicked on it. Therefore, the idea was to make a component that could have anything inside it, and react wouldn't keep trying to remove it. It's a limitation of using a third party library which appends components into the DOM. Does that make sense?

Reply:
If you don't supply container, then it defaults to body, which is fine normally, but it will fire a blur which will trigger focus outside. That's why I'm supplying container ... to keep the focus inside the post title block.